frontend/src/components/search/SearchFilters.tsx

```diff
@@ -23,7 +22,6 @@ export default function SearchFilters({
   category: Set<string>;
   opportunityStatus: Set<string>;
 }) {
-  unstable_setRequestLocale("en");
```

Comment:
TIL that `unstable_setRequestLocale` is only meant to be used on layout and page files https://next-intl-docs.vercel.app/docs/getting-started/app-router/with-i18n-routing#add-unstable_setrequestlocale-to-all-layouts-and-pages